src/Components/Assets/AssetFilter.tsx

```diff
@@ -18,9 +18,42 @@ import { dateQueryString } from "../../Utils/utils";
 const getDate = (value: any) =>
   value && dayjs(value).isValid() && dayjs(value).toDate();
 
+const initialFacilityState: FacilityModel = {
+  id: undefined,
+  name: "",
+  read_cover_image_url: "",
+  facility_type: "",
+  address: "",
+  features: [],
+  location: {
+    latitude: 0,
+    longitude: 0,
+  },
+  oxygen_capacity: 0,
+  phone_number: "",
+  type_b_cylinders: 0,
+  type_c_cylinders: 0,
+  type_d_cylinders: 0,
+  middleware_address: "",
+  expected_type_b_cylinders: 0,
+  expected_type_c_cylinders: 0,
+  expected_type_d_cylinders: 0,
+  expected_oxygen_requirement: 0,
+  local_body_object: undefined,
+  district_object: undefined,
+  state_object: undefined,
+  ward_object: undefined,
+  modified_date: "",
+  created_date: "",
+  state: 0,
+  district: 0,
+  local_body: 0,
+  ward: 0,
+};
+
 function AssetFilter(props: any) {
   const { filter, onChange, closeFilter } = props;
-  const [facility, setFacility] = useState<FacilityModel>({ name: "" });
+  const [facility, setFacility] = useState<FacilityModel>(initialFacilityState);
```

Comment:
Do not have a placeholder if there is no facility data present. instead use undefined.

```suggestion
  const [facility, setFacility] = useState<FacilityModel>();
```